app/assets/stylesheets/navbar.css.scss

```diff
@@ -0,0 +1,59 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@400;700&display=swap');
+
+$primaryColor: #AEFEFF;
+$very_pale_cyan: #4FBDBA;
+$very_Dark_Cyan: #072227;
+$normal_Cyan: #35858B;
+
+.navbar {
+  clear: both;
+  overflow: auto;
+  background-color: $very_Dark_Cyan;
+  border-radius: 50px;
+  font-family: 'Quicksand', sans-serif;
+  color: white;
```

Comment:
Please, sort this block alphabetically, in VS code can be sort with `ctrl + shift + P` and selec the option `sort lines ascending `

Reply:
@JuanequeX solved!